FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals.rdf

```diff
@@ -0,0 +1,2119 @@
+		<fibo-fnd-utl-av:preferredDesignation>BYU</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Beach">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>beach</rdfs:label>
+		<skos:definition>shore of a body of water covered by sand, gravel, or larger rock fragments</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BCH</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BEACH</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BCH</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Bend">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>bend</rdfs:label>
+		<skos:definition>curve, especially a sharp one, in a road, river, racecourse, or path</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BEND</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BND</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BND</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Bluff">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>bluff</rdfs:label>
+		<skos:definition>broad, rounded cliff</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BLF</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BLUF</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BLUFF</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BLF</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Bluffs">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>bluffs</rdfs:label>
+		<skos:definition>multiple broad, rounded cliffs</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BLUFFS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BLFS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Bottom">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>bottom</rdfs:label>
+		<skos:definition>deepest or lowest part of something, such as the bottom of a hill</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BOT</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BOTTM</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BOTTOM</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BTM</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BTM</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Boulevard">
```

Comment:
A lot of these are not US-specific. How would we cope with a France specific file without breaking the unique names rule?